docs/GettingStarted.md

```diff
@@ -165,6 +180,25 @@ Install and configure the Datadog Agent to receive traces from your now instrume
 
 ### Quickstart for Ruby applications
 
+#### Ruby Auto Instrument all Integrations
```

Comment:
Should we not say here "Auto Instrument All Integrations". That's a bit wordy and inconsistent with how we describe in other languages. Maybe just "Automatic Instrumentation" and "Manual Instrumentation"

Reply:
It reads much better with your suggestion, probably an oversight on our side here. I'll make these changes. 🙇  @andrewsouthard1